src/felles/komponenter/rekbis-kort/RekBisKort.tsx

```diff
@@ -0,0 +1,23 @@
+import { Panel } from '@navikt/ds-react';
+import classNames from 'classnames';
+import * as React from 'react';
+import css from './RekBisKort.module.css';
+
+export interface IRekBisKort {
+    fremhevet?: boolean;
+    header?: React.ReactNode | undefined;
+    children?: React.ReactNode | undefined;
+    footer?: React.ReactNode | undefined;
+}
+
+const RekBisKort: React.FC<IRekBisKort> = ({ header, children, footer, fremhevet }) => {
```

Comment:
React.FC/FunctionComponent er anbefalt å ikke bruke lenger. For nye komponenter pleier vi nå å si

```
const Komponent = ({ prop1, prop2 }: Props) => {
   ...
}
``` 